sparse/core.py

```diff
@@ -111,6 +111,15 @@ def __init__(self, coords, data=None, shape=None, has_duplicates=True,
                 coords = list(coords.items())
                 has_duplicates = False
 
+            if isinstance(coords, COO):
+                self.coords = coords.coords
+                self.data = coords.data
+                self.has_duplicates = coords.has_duplicates
+                self.sorted = coords.sorted
+                self.shape = coords.shape
+                return
+
```

Comment:
`flake8` is crapping out here. You can have a maximum of one blank line inside of a function.